src/Sarif/Comparers/CodeFlowComparer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+/// <summary>
+/// Note: This comparer may not have all properties compared. Will be replaced by comprehensive
+/// comparer generated by JSchema as part of EqualityComparer in a planned comprehensive solution.
+/// Tracking by issue: https://github.com/microsoft/jschema/issues/141
+/// </summary>
+namespace Microsoft.CodeAnalysis.Sarif.Comparers
+{
+    internal class CodeFlowComparer : IComparer<CodeFlow>
+    {
+        internal static readonly CodeFlowComparer Instance = new CodeFlowComparer();
```

Comment:
This should be public if we follow the existing pattern and this declaration would actually be moved to the class in question, in this example 'CodeFlow'.